src/other/tables.jl

```diff
@@ -43,7 +43,7 @@ fromcolumns(x, names; copycols::Bool=true) =
               copycols=copycols)
 
 function DataFrame(x::T; copycols::Bool=true) where {T}
-    if !Tables.istable(x)
+    if !Tables.istable(T)
```

Comment:
Is this just to work around the mentioned bug?

Reply:
No - I just think it is cleaner to dispatch on type and we it available as this is the "intended" way to use `istable` (but in particular - while we have the bug - it resolves it)